source/Calamari/Program.cs

```diff
@@ -107,13 +107,25 @@ public static void EnableAllSecurityProtocols()
             // TLS1.1 and below was discontinued on MavenCentral as of 18 June 2018
             //https://central.sonatype.org/articles/2018/May/04/discontinue-support-for-tlsv11-and-below/
 
-            ServicePointManager.SecurityProtocol =
+            var securityProcotolTypes =
 #if !NETCOREAPP2_0
                 SecurityProtocolType.Ssl3 |
 #endif
-                SecurityProtocolType.Tls
-                | (SecurityProtocolType) 768 // TLS1.1
-                | (SecurityProtocolType) 3072; // Unfortunately SecurityProtocolType.TLS12 doesn't exist in net40 but its enum value works!
+                SecurityProtocolType.Tls;
+
+            if (Enum.IsDefined(typeof(SecurityProtocolType), 768))
+                securityProcotolTypes = securityProcotolTypes | (SecurityProtocolType)768;
+
+            if (Enum.IsDefined(typeof(SecurityProtocolType), 3072))
+            {
+                securityProcotolTypes = securityProcotolTypes | (SecurityProtocolType)3072;
+            }
+            else
+            {
+                Log.Verbose($"TLS1.2 is not supported, this means that some outgoing connections to third party endpoints will not work as they now only support TLS1.2.{Environment.NewLine}This includes GitHub feeds and Maven feeds.");
```

Comment:
Is this worth logging as warning? or at least info?

Reply:
I dont think warning is great if for 99% of use cases it wont be a problem. When their script _does_ fail, at least we\they can get some info about this in the logs